test/test_nn.py

```diff
@@ -12366,6 +12366,14 @@ def test_embedding_padding_idx(self, device, dtype):
         self.assertEqual(output[0][2].sum(), 0)
         self.assertEqual(output[1][1].sum(), 0)
 
+        # change padding vector
+        padding_vector = torch.ones(20, dtype=dtype)
+        embedding = nn.Embedding(10, 20, padding_idx=2, sparse=True).to(device, dtype)
+        embedding.weight.data[2] = padding_vector
```

Comment:
An idea: to add a `.padding_embedding` property on `nn.Embedding`.
It'd be more intuitive to do `embedding.padding_embedding.copy_(padding_vector)` than `embedding.weight.data[2] = padding_vector`? Hmm, although we couldn't make use of assignment operator in that case? ;/